### PR TITLE
[INFRA-1210] Accountapp: Add Auth options for SMTP server

### PR DESCRIPTION
**Pull request by olblak**
* Update README with SMTP configurations options
     * Update entrypoint.sh with new SMTP configurations
     * Add username,password with starttls authentication on port 587 for SMTP server

### CODE REVIEW

**Comment by rtyler** [comment]
@daniel-beck, as a resident Java developer in the infra group, would you mind taking a look at t his? :smiley_cat:

File: src/main/java/org/jenkinsci/account/Application.java [hunk 1]
```diff
@@ -487,9 +488,24 @@ public void delete(DirContext con) throws NamingException {
     }
 
     private Session createJavaMailSession() {
+        Session session;
         Properties props = new Properties(System.getProperties());
+        System.out.printf(params.smtpAuth());
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/account/Application.java]: Leftover debug code?
Also, no trailing newline.

File: src/main/java/org/jenkinsci/account/Application.java [hunk 2]
```diff
@@ -487,9 +488,24 @@ public void delete(DirContext con) throws NamingException {
...
         props.put("mail.smtp.host",params.smtpServer());
-        return Session.getInstance(props);
+        if(params.smtpAuth().equals("true")) {
+            props.put("mail.smtp.auth", params.smtpAuth());
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/account/Application.java]: This seems to be a boolean valued String, so `String` seems errorprone.

**Comment by daniel-beck** [comment]
Looks good overall.

Note I have only read this, not run it.

File: src/main/java/org/jenkinsci/account/Application.java (hunk 1 above)
Comment (olblak) [inline src/main/java/org/jenkinsci/account/Application.java]: Indeed, I forgot to remove debug code. I fix that

File: src/main/java/org/jenkinsci/account/Application.java (hunk 2 above)
Comment (olblak) [inline src/main/java/org/jenkinsci/account/Application.java]: Indeed it's error prone and doesn't make sense, I change this for a Boolean

**Pull request merged**